packages/nimbus/src/components/base/containers/ResponseContainer.tsx

```
@@ -1,9 +1,19 @@
-    <div>ResponseContainer</div>
-  )
+	const mockApiResponseData = useSelector(selectMockApiResponseData)
+	return (
+		<div
+			className='bg-yellow-500 h-full overflow-auto'
+			style={{
+				border: 'var(--border-width) solid var(--color-border)',
+				borderRadius: 'var(--border-radius)'
+			}}>
+			ResponseContainer
+			<>{JSON.stringify(mockApiResponseData?.data?.data)}</>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)